www/views/page/content/list.php

```diff
@@ -43,13 +68,21 @@ function cap($text, $len)
 $n = 0;
 foreach ($articles as $article) {
     $filtered = false;
+    if (!empty($catFilter)) {
+	$filtered = $filtered || !(in_array($article->category_id, $catFilter));
+    }
     if (isset($filter)) {
         if ($filter >= 1) {
             $filtered = $filtered || $article->ready === false;
         }
-        if ($filter >= 2) {
-            $filtered = $filtered || count($article->checkers) > 0;
+        if ($filter == 2) {
+	    $user =  Session::instance()->getUser();
+            $filtered = $filtered || count($article->checkers) > 2 || in_array($user, $article->checkers) ||  in_array($user, $article->authors);
```

Comment:
Dit is wel net wat anders dan de beschrijving voor de gebruiker

Reply:
Ik kan de beschrijving veranderen naar "klaar & kan ik nakijken", want hier krijg je alle stukjes die klaar zijn en die nog nagekeken moeten worden en ook door jou nog nagekeken kunnen worden, want als jij het stukje hebt geschreven of al hebt nagekeken kan je het stukje niet meer nakijken.